cloudinit/distros/__init__.py

```diff
@@ -1248,6 +1248,21 @@ def build_dhclient_cmd(
             "/bin/true",
         ] + (["-cf", config_file, interface] if config_file else [interface])
 
+    @staticmethod
+    def eject_media(device: str) -> None:
+        cmd = None
+        if subp.which("eject"):
```

Comment:
i just checked, and this code path will work on OpenBSD, NetBSD, Dragonfly(!), and not on FreeBSD.

so I'll need to submit a patch

Reply:
https://github.com/catmsred/cloud-init/pull/1

Reply:
Per @igalic's suggestion should we add camcontrol to the main distro list of ejects to try?  While it's currently only effecting FreeBSD, there may come a time when eject is removed from some of the other distros too.

Reply:
CAM is a FreeBSD thing, so I don't see a reason to apply it everywhere. I'd rather merge this first and then take the other PR separately.